examples/jumper.js

```diff
@@ -5,14 +5,36 @@
  *
  * This bot can move, jump, ride vehicles, attack nearby entities and much more.
  */
-const mineflayer = require('mineflayer')
+import { createBot } from 'mineflayer'
 
 if (process.argv.length < 4 || process.argv.length > 6) {
-  console.log('Usage : node jumper.js <host> <port> [<name>] [<password>]')
+  console.log('Usage : node trader.js <host> <port> [<name>] [<password>]')
   process.exit(1)
 }
 
-const bot = mineflayer.createBot({
+console.log('Commands :\n' +
+  '  forward\n' +
+  '  back\n' +
+  '  left\n' +
+  '  right\n' +
+  '  sprint\n' +
+  '  stop\n' +
+  '  jump\n' +
+  '  jump a lot\n' +
+  '  stop jumping\n' +
+  '  attack\n' +
+  '  mount\n' +
+  '  dismount\n' +
+  '  move vehicle forward\n' +
+  '  move vehicle backward\n' +
+  '  move vehicle left\n' +
+  '  move vehicle left\n' +
+  '  move vehicle right\n' +
+  '  tp\n' +
+  '  pos\n' +
+  '  yp')
```

Comment:
Why console log? It's not intended to be used as a bot. It's intended to act as an example of a bot. Anyone who use that file will look at the code before running it.